### PR TITLE
ZCS-7775: Add gql automation for admin get account and delete account mutations

### PR DESCRIPTION
**Pull request by ajinkya81**
Add gql automation for admin get account and delete account mutations
Please unzip attached report and open index.html to view report.
[target.zip](https://github.com/Zimbra/zm-api-test-harness/files/3559195/target.zip)

### CODE REVIEW

File: src/java/stepDefinitions/AccountStepdefs.java [hunk 1]
```diff
@@ -105,11 +123,47 @@ public void modifyAccount(String attrList, String name){
...
+        HttpResponse response = baseline.processRequest(context, requestBody, HttpPost.METHOD_NAME);
+        context.setResponse(response);
+    }
+
+    @Given("^Create and Get user with name '(.+)' password '(.+)' attribute '(.+)'$")
+    public void getAccount(String name, String password, String attrList) {
+        this.createAccount(name, password, attrList);
+        String accountNamePath = "data.accountCreate.account.name";
+        String accountNameValue = baseline.getValue(context, accountNamePath);
+        scenario.write("Actual value in the response: "+accountNameValue);
```
Comment (dasiyogesh) [inline src/java/stepDefinitions/AccountStepdefs.java]: add space around '+' or can use String.format

File: src/java/stepDefinitions/AccountStepdefs.java [hunk 2]
```diff
@@ -52,27 +56,41 @@ public void createAccount(String name, String password, String attrList) {
...
+            String jsonPath = accountAttrPath + "[?(@.key=='"+attribute+"')].value";
             String actValue = baseline.getValue(context, jsonPath);
             scenario.write("Actual value in the response: "+actValue);
             scenario.write("Expected value: [\""+expValue+"\"]");
             Assert.assertEquals("[\""+expValue+"\"]", actValue);
         }
     }
 
+    @Then("^Validate account is deleted$")
+    public void validateAccountDelete(){
```
Comment (dasiyogesh) [inline src/java/stepDefinitions/AccountStepdefs.java]: add space before opening {

File: src/java/stepDefinitions/AccountStepdefs.java (hunk 1 above)
Comment (ajinkya81) [reply src/java/stepDefinitions/AccountStepdefs.java]: fixed

File: src/java/stepDefinitions/AccountStepdefs.java (hunk 2 above)
Comment (ajinkya81) [reply src/java/stepDefinitions/AccountStepdefs.java]: fixed

**Pull request merged**